deploy/operator.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           image: $SRIOV_NETWORK_OPERATOR_IMAGE
           command:
           - sriov-network-operator
+          args:
+          - --leader-elect=$OPERATOR_LEADER_ELECTION_ENABLE
```

Comment:
do we need the also in the config folder,helm folder and for the openshift bundle?

Reply:
openshift bundle already has the `--leader-elect=true` parameter

https://github.com/openshift/sriov-network-operator/blob/master/manifests/stable/sriov-network-operator.clusterserviceversion.yaml#L335

I don't know if we want to enable this feature for Helm, too. Maybe we can discuss and implement it in a different PR?

Reply:
sounds good can you please open an issue so we don't forget :)